composite-products-conditional-images-for-woocommerce.php

```diff
@@ -114,10 +114,8 @@ public static function load_plugin() {
 		// Add qty data in scenarios.
 		add_filter( 'woocommerce_composite_current_scenario_data', array( __CLASS__, 'scenario_data' ), 10, 4 );
 
-		if ( version_compare( WC_CP()->version, '8.0' ) < 0 ) {
```

Comment:
Looks like this should have been disabled for Composite Products versions earlier than v8, not enabled.